### PR TITLE
[Doc] add terms for site search to return pages correctly for bool and range queries

### PR DESCRIPTION
**Pull request by alicejw-aws**
### Description
bool and range query - minor - add terms for site search to return pages correctly

### Related issues
Site search usability issue is related: [Search needs relevance tuning[(https://github.com/opensearch-project/documentation-website/issues/1117)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/query-dsl/term.md [hunk 1]
```diff
@@ -310,6 +310,9 @@ Parameter | Behavior
 `lte` | Less than or equal to.
 `lt` | Less than.
 
+In addition to the range query parameters, you can provide date formats or relation operators such as "contains" or "within." To see the supported field types for range queries, see [Range query optional parameters]({{site.url}}{{site.baseurl}}/opensearch//supported-field-types/range/#range-query-optional-parameters).
```
Comment (Naarcha-AWS) [inline _opensearch/query-dsl/term.md:313]: Do we have a link to these date formats? Maybe to here, https://opensearch.org/docs/latest/opensearch/supported-field-types/date/#formats?

**Review by Naarcha-AWS — APPROVED** [review]
Small suggestion, otherwise LGTM.

File: _opensearch/query-dsl/bool.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ nav_order: 45
 
 # Boolean queries
 
-The `bool` query lets you combine multiple search queries with boolean logic. You can use boolean logic between queries to either narrow or broaden your search results.
+You can perform a boolean query with the `bool` query type. A boolean query compounds query clauses so you can combine multiple search queries with boolean logic. To narrow or broaden your search results, use the `bool` query clause rules.
```
Comment (kolchfa-aws) [inline _opensearch/query-dsl/bool.md:10]: Capitalize "Boolean" per our style guide

**Review by kolchfa-aws — APPROVED** [review]
Looks great.

**Pull request merged**